bciers/apps/administration/app/bceidbusiness/industry_user/select-operator/(request-access)/page.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import SelectOperatorForm from "../../../../components/userOperators/SelectOperatorForm";
+import { useSession } from "next-auth/react";
+import getUserFullName from "@bciers/utils/getUserFullName";
+export default function Page() {
+  const { data: session } = useSession();
+  const names = getUserFullName(session)?.split(" ");
+
+  return (
+    <>
+      <section className="text-center my-auto text-2xl flex flex-col gap-3 mx-auto">
+        <p>
+          Hi,{" "}
+          <b>
+            {names?.[0]} {names?.[1]}!
+          </b>
+        </p>
+        <p>Which operator would you like to log in to?</p>
```

Comment:
I'm sure this fits with the design though `Which operator would you like to log in to?` sounds confusing somehow.

Reply:
@marcellmueller 

Agreed but...

![image](https://github.com/user-attachments/assets/b79ba339-492d-4653-8b0d-5f105d0cdf44)